test-kernel/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2018"
 
 [dependencies]
 x86_64 = "0.3.4"
+bootloader = {path = "..", features=["sse"]}
```

Comment:
I'm not sure if this is a good way to test it because we no longer test it without the `sse` feature this way. Instead we should create multiple test kernels with different feature combinations like we do for [bootimage](https://github.com/rust-osdev/bootimage). This does not need to be part of this PR, though.

I think the best way forward is to merge this without tests (undoing the modifications to the `test-kernel`) and add proper tests in a follow-up pull request.